### ISSUE TITLE
[Screen Reader-Home Page]: Default Lang Attribute is not defined for the Home Page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2646]
**User Experience:**
If Default Lang Attribute is not defined for any web page then it would be difficult for the Screen reader users to access that Web page.

**Test Environment:**
OS: Windows 10 RS2
Build: 15063.2
Browser: Edge
Tool Used: IE11 Developer Tool

**Steps to reproduce:**
1. Navigate to url:- https://makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3-Verify, the Lang attribute of the Hoem Page using F12

**Actual Behavior:**
Default Lang Attribute is not defined for the Home Page.

**Expected Behavior:**
Default Lang Attribute should be defined for the Home Page..

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={eb8515c6-18d7-4af0-b47e-c46958b1b50e}

**Suggested Fix:**
1- Use the title attribute. This technique is most often used when there is not space in the design for an explicit text label.  
e.g. 
<input type="text" title="Type search term here"/> <input type="submit" value="Search"/>  
 
2-Use the fieldset and legend elements to provide a description for groups of form elements. This is used for more complex forms and is often used in conjunction with other techniques. 
 
3-Use "aria-label". This technique is an alternative to the title attribute technique above in point 1.  
e.g. 
<button aria-label="Close" onclick="myDialog.close()">X</button> 
 
4-Use the "aria-describedby" property to provide a descriptive label for user interface controls 
e.g. 
<button aria-label="Close" aria-describedby="descriptionClose" 
    onclick="myDialog.close()">X</button> 
… 
<div id="descriptionClose">Closing this window will discard any information entered and 
return you back to the main page</div> 
 
5- Use 'aria-labelledby' to concatenate a label from several text nodes 

6-Providing text descriptions to identify required fields that were not completed 
 Key References for Fix: 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html 

**Please refer the Attachment:**
![mas3 1 1_home page](https://user-images.githubusercontent.com/30434417/28908438-4c7682b8-7841-11e7-8085-c2455fd37915.jpg)
1- Use the title attribute. This technique is most often used when there is not space in the design for an explicit text label.  
e.g. 
<input type="text" title="Type search term here"/> <input type="submit" value="Search"/>  
 
2-Use the fieldset and legend elements to provide a description for groups of form elements. This is used for more complex forms and is often used in conjunction with other techniques. 
 
3-Use "aria-label". This technique is an alternative to the title attribute technique above in point 1.  
e.g. 
<button aria-label="Close" onclick="myDialog.close()">X</button> 
 
4-Use the "aria-describedby" property to provide a descriptive label for user interface controls 
e.g. 
<button aria-label="Close" aria-describedby="descriptionClose" 
    onclick="myDialog.close()">X</button> 
… 
<div id="descriptionClose">Closing this window will discard any information entered and 
return you back to the main page</div> 
 
5- Use 'aria-labelledby' to concatenate a label from several text nodes 

6-Providing text descriptions to identify required fields that were not completed 
 Key References for Fix: 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html 

**Please refer the Attachment:**
![mas3 1 1_home page](https://user-images.githubusercontent.com/30434417/28908438-4c7682b8-7841-11e7-8085-c2455fd37915.jpg)

**Comment by microsoftsam** [comment]
Fixed in makecode.com

**Comment by v-mesala** [comment]
Bug is verified and fixed .Hence closed the bug.
